recipes/stx/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.8)
+project(PackageTest LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(example_basic example.cpp)
+target_link_libraries(example_basic PRIVATE CONAN_PKG::stx)
+target_compile_options(example_basic PRIVATE $<$<CXX_COMPILER_ID:MSVC>:/permissive->)
```

Comment:
Should it be added in `cpp_info.cxxflags` if msvc?

Reply:
I think it's up to consumers to enforce `/permissive-` since it can affect ABI

Reply:
Sorry, /permissive- is not actually required by the library.
I added it out of habit, to check that moderately heavy template code still works in "strict" mode,
but the library is usable without the flag, so it shouldn't be forced in the test_package script.